package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.3",
   "homepage": "https://github.com/aframevr/aframe-core",
   "license": "MIT",
-  "main": "src/index.js",
+  "main": "dist/aframe-core.js",
```

Comment:
I've updated the README in https://github.com/aframevr/aframe/pull/362

We're going to work on merging repositories tomorrow. Moving all development and issues over.

This PR works to make it so the library is requirable without extra work.

Not as worried about documenting `npm link` as that won't be too commonly done.